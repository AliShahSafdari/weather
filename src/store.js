import { configureStore } from '@reduxjs/toolkit';
import weathersReducer from './redux/weather/weathersReducer';

const store = configureStore({
  reducer: {
    weathers: weathersReducer,
  },
});

export default store;
