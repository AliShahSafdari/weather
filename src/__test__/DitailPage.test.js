import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import CityDetails from '../components/DitailPage';
import store from '../store';

describe('Component render testing', () => {
    it('Home Page perfectly', () => {
        const { asFragment } = render(
            <Provider store={store}>
                <CityDetails />
            </Provider>,
        );
        expect(asFragment()).toMatchSnapshot();
    });
});