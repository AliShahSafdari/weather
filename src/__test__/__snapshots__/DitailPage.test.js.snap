// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component render testing Home Page perfectly 1`] = `
<DocumentFragment>
  <h1>
    City Details
  </h1>
</DocumentFragment>
`;
